README.md

```diff
@@ -50,6 +56,18 @@ For issues see [Snovault OSX App Installation][] first.
     $ ./circle-tests.sh npm
     ```
 
+You can also invoke `pytest` directly if you need more granular control over which Python tests to run.
+
+```bash
+# Make sure you are in your venv
+# Run a specific test in a specific file
+$ pytest TEST_FILE_PATH::TEST_NAME
+# Run tests with a name matching the expression
+$ pytest -k $EXPRESSION
```

Comment:
What does expression mean here? So if I would have `test_foo_1` and `test_foo_2` then `pytest -k test_foo_*` would run both of those?

Reply:
It is based on substring matching + python operators. To run both you would use `-k test_foo`. However you can have expressions like `not test_bar`, `foo and bar`, etc. I agree it's not clear what expression is supposed to be, let me add a link to the pytest docs.